BP/common/scripted_effects/00_scripted_effects.txt

```diff
@@ -0,0 +1,564 @@
+﻿
```

Comment:
It is unclear what you changed in this file?

Reply:
Nothing apparently; another holdover file.